CRM/ACL/BAO/ACL.php

```diff
@@ -160,11 +160,10 @@ protected static function getACLs(int $contact_id) {
 
     $rule = new CRM_ACL_BAO_ACL();
 
-    $acl = self::getTableName();
     $contact = CRM_Contact_BAO_Contact::getTableName();
 
     $query = " SELECT acl.*
```

Comment:
@seamuslee001 I've been digging on this - I can't find any evidence that anywhere in core ever saves to civicrm_acl with the value entity_table = 'contact'

It seems the OG bridge has some hard coding to acl_role